content/blog/2020-04-30-table/index.Rmd

```diff
@@ -0,0 +1,132 @@
+knitr::knit_hooks$set(
+  plot = function(x, options) {
+    hugoopts <- options$hugoopts
+    paste0(
+      "{{<figure src=",
+      '"', x, '" ',
+      if (!is.null(hugoopts)) {
+        glue::glue_collapse(
+          glue::glue('{names(hugoopts)}="{hugoopts}"'),
+          sep = " "
+        )
+      },
+      ">}}\n"
+    )
+  }
+)
+
+# knitr hook to use Hugo highlighting options
+knitr::knit_hooks$set(
+  source = function(x, options) {
+  hlopts <- options$hlopts
+    paste0(
+      "```r ",
+      if (!is.null(hlopts)) {
+      paste0("{",
+        glue::glue_collapse(
+          glue::glue('{names(hlopts)}={hlopts}'),
+          sep = ","
+        ), "}"
+        )
+      },
+      "\n", glue::glue_collapse(x, sep = "\n"), "\n```\n"
+    )
+  }
+)
+```
+
+
+
+Cat ipsum dolor sit amet, kick up litter for man running from cops stops to pet cats, goes to jail, i shredded your linens for you kitty run to human with blood on mouth from frenzied attack on poor innocent mouse, don't i look cute?. I hate cucumber pls dont throw it at me intently sniff hand. Leave hair on owner's clothes bite the neighbor's bratty kid. And sometimes switches in french and say "miaou" just because well why not lick plastic bags. Cat ass trophy meowing chowing and wowing but i shredded your linens for you, so crash against wall but walk away like nothing happened. Do i like standing on litter cuz i sits when i have spaces, my cat buddies have no litter i live in luxury cat life purrrrrr or cough hairball on conveniently placed pants or kitty kitty swat at dog make meme, make cute face so cat milk copy park pee walk owner escape bored tired cage droppings sick vet vomit. Slap owner's face at 5am until human fills food dish spend six hours per day washing, but still have a crusty butthole. Ask for petting intently sniff hand, and gimme attention gimme attention gimme attention gimme attention gimme attention gimme attention just kidding i don't want it anymore meow bye so chase dog then run away. Have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat let me in let me out let me in let me out let me in let me out who broke this door anyway the fat cat sat on the mat bat away with paws and have a lot of grump in yourself because you can't forget to be grumpy and not be like king grumpy cat furrier and even more furrier hairball. Jump launch to pounce upon little yarn mouse, bare fangs at toy run hide in litter box until treats are fed fooled again thinking the dog likes me but fall over dead (not really but gets sypathy). Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff massacre a bird in the living room and then look like the cutest and most innocent animal on the planet and sweet beast. Climb leg.[^cat]
+
+
+
+```{r table, echo=FALSE, message=FALSE}
+packages <- readr::read_tsv("data-access-pkgs.tsv")
```

Comment:
I haven't added these two TSV in the PR, that's the Google sheet and [the citations file](https://github.com/ropenscilabs/ropensci_citations/blob/master/citations_all.tsv).